index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="main.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+    <title>Medical core</title>
+</head>
+<body class="container">
+        <nav class="navbar">
+        <h1 class="navbar__logo">Medical core</h1>
+        <ul class="menu">
+            <li class="menu__item">Home</li>
+            <li class="menu__item">Product</li>
+            <li class="menu__item">Pricing</li>
+            <li class="menu__item">Contact</li>
+        </ul>
+        <ul class="auth">
+            <li class="auth__login">Login</li>
+            <li class="auth__signup button">JOIN US <span class="material-symbols-outlined">
```

Comment:
Instead of using button class, I suggest you use a modifier. Also I suggest you create a btn block to be reused on all buttons